go/vt/tabletserver/tabletconn/tablet_conn.go

```diff
@@ -76,6 +76,11 @@ type TabletConn interface {
 	Commit(context context.Context, transactionId int64) error
 	Rollback(context context.Context, transactionId int64) error
 
+	// These should not be used for anything except tests for now; they will eventually
+	// replace the existing methods.
+	UnsupportedNewCommit(context context.Context, transactionId int64) error
```

Comment:
It is better to name these as Commit2, and leave it untouched in future.
Otherwise, you need to modify the interface again.